### ISSUE TITLE
ARIA 12 needs updating to support the ARIA 1.2 requirement that aria-level must be specified.

### ISSUE THREAD

**Issue by fstrr** [w3c/wcag#3733]
The ARIA 1.2 specification for [role="heading"](https://www.w3.org/TR/wai-aria/#heading) requires that an [aria-level](https://www.w3.org/TR/wai-aria-1.3/#aria-level) property is also used, which is a change from the ARIA 1.1 spec, where a missing aria-level is treated as a second-level heading (an `h2`).

The [WCAG 2.2 version of ARIA 12](https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA12) doesn't mention this requirement in its examples, so needs updating.

The [draft ARIA 1.3 spec for aria-level](https://www.w3.org/TR/wai-aria-1.3/#aria-level) includes a useful note around aria-level values of 10 and higher, which would be good for WCAG to also mention.
